### ISSUE TITLE
doctest-prop-0.2 test suite failure

### ISSUE THREAD

**Issue by peti** [nushio3/doctest-prop#1]
Citing from http://hydra.cryp.to/build/785186/nixlog/1/raw:

```
Test suite test: RUNNING...
### Failure in src/Test/DocTest/Prop.hs:15: expression `propWith (quietArgs{maxSize=3}) $ (<10).length'
expected: 
 but got: 
          <interactive>:26:1:
              No instance for (Show (t0 a0)) arising from a use of ‘propWith’
              The type variables ‘t0’, ‘a0’ are ambiguous
              Note: there are several potential instances:
                instance Show a => Show (Data.Complex.Complex a)
                  -- Defined in ‘Data.Complex’
                instance (Show a, Show b) => Show (Either a b)
                  -- Defined in ‘Data.Either’
                instance Data.Fixed.HasResolution a => Show (Data.Fixed.Fixed a)
                  -- Defined in ‘Data.Fixed’
                ...plus 19 others
              In the expression: propWith (quietArgs {maxSize = 3})
              In the expression:
                propWith (quietArgs {maxSize = 3}) $ (< 10) . length
              In an equation for ‘it’:
                  it = propWith (quietArgs {maxSize = 3}) $ (< 10) . length

          <interactive>:26:41:
              No instance for (Foldable t0) arising from a use of ‘length’
              The type variable ‘t0’ is ambiguous
              Note: there are several potential instances:
                instance Foldable (Either a) -- Defined in ‘Data.Foldable’
                instance Foldable Data.Proxy.Proxy -- Defined in ‘Data.Foldable’
                instance GHC.Arr.Ix i => Foldable (GHC.Arr.Array i)
                  -- Defined in ‘Data.Foldable’
                ...plus three others
              In the second argument of ‘(.)’, namely ‘length’
              In the second argument of ‘($)’, namely ‘(< 10) . length’
              In the expression:
                propWith (quietArgs {maxSize = 3}) $ (< 10) . length
Examples: 5  Tried: 4  Errors: 0  Failures: 1
Test suite test: FAIL
```

**Comment by peti** [comment]
Ping?

**Comment by nushio3** [comment]
I am sorry, but `doctest-prop` library has been deprecated in favor of [QuickCheck feature of doctest](https://github.com/sol/doctest#quickcheck-properties). I would take this opportunity to learn how to deprecate a package. Thank you for paying attention on, and I am sorry for your disappointment if you would like to use, my `doctest-prop` library.

**Comment by nushio3** [comment]
Closed by  5d12ec1ae00f21494754dc4422afd .
